### PR TITLE
Valkiria commands

### PR DESCRIPTION
**Pull request by eruizgar**
- Command to start Valkiria process. If Valkiria is running give and info message.
- Command to stop Valkiria process. If Valkiria is stopped give and info message.
- Command to status Valkiria process. Give an info message that tell if Valkiria agent is running or is stopped.
- Command to uninstall Valkiria. If Valkiria is running stopped and remove the directories.
- Fix problem when install Valkiria and is already installed including a pid file that help to check the status of Valkiria in every moment.

### CODE REVIEW

File: dcos_valkiria/cli.py [hunk 1]
```diff
@@ -16,14 +20,41 @@
...
+    dcos valkiria status [--option SSHOPT=VAL ...]
+            [--config-file=<path>]
+            [--user=<user>]
+            (--ips=<"ip-or-iplist">)
 
 Commands:
     install
         Establish an SSH connection to the master or agent nodes and install
         valkiria in the ip passed.
+    uninstall
```
Comment (adoblas) [inline dcos_valkiria/cli.py]: @albertostratio Can you please review help info?

File: dcos_valkiria/cli.py [hunk 2]
```diff
@@ -16,14 +20,41 @@
...
+        Establish an SSH connection to the master or agent nodes and uninstall
+        valkiria in the ip passed.
```
Comment (albertostratio) [inline dcos_valkiria/cli.py]: I'd say: "Establish an SSH connection to the master or agent nodes and uninstall Valkiria on them"

File: dcos_valkiria/cli.py [hunk 3]
```diff
@@ -16,14 +20,41 @@
...
     tasks
         Establish an SSH connection to the master or agent nodes and list
         the killables tasks.
+    kill
+        Establish an SSH connection to the master or agent nodes and kill
+        the task to a particular taskId.
```
Comment (albertostratio) [inline dcos_valkiria/cli.py]: I'd say: "Establish an SSH connection to the master or agent nodes and kill the task with a specific taskId."

File: dcos_valkiria/cli.py [hunk 4]
```diff
@@ -16,14 +20,41 @@
...
+            [--user=<user>]
+            (--ips=<"ip-or-iplist">)
+    dcos valkiria status [--option SSHOPT=VAL ...]
+            [--config-file=<path>]
+            [--user=<user>]
+            (--ips=<"ip-or-iplist">)
 
 Commands:
     install
         Establish an SSH connection to the master or agent nodes and install
```
Comment (albertostratio) [inline dcos_valkiria/cli.py]: I'd say: "Establish an SSH connection to the master or agent nodes and install Valkiria on them."

File: dcos_valkiria/cli.py [hunk 5]
```diff
@@ -16,14 +20,41 @@
...
+        Establish an SSH connection to the master or agent nodes and uninstall
+        valkiria in the ip passed.
     tasks
         Establish an SSH connection to the master or agent nodes and list
         the killables tasks.
+    kill
+        Establish an SSH connection to the master or agent nodes and kill
+        the task to a particular taskId.
+    start
+        Establish an SSH connection to the master or agent nodes and list
```
Comment (albertostratio) [inline dcos_valkiria/cli.py]: I'd say: "Establish an SSH connection to the master or agent nodes and start the Valkiria agent process on them."

File: dcos_valkiria/cli.py [hunk 6]
```diff
@@ -16,14 +20,41 @@
...
+        and start the valkiria agent process.
+    stop
+        Establish an SSH connection to the master or agent nodes and list
+        and stop the valkiria agent process.
```
Comment (albertostratio) [inline dcos_valkiria/cli.py]: I'd say: "Establish an SSH connection to the master or agent nodes and stop the Valkiria agent process on them."

File: dcos_valkiria/cli.py [hunk 7]
```diff
@@ -16,14 +20,41 @@
...
+    stop
```
Comment (albertostratio) [inline dcos_valkiria/cli.py]: Should be status

File: dcos_valkiria/cli.py [hunk 8]
```diff
@@ -16,14 +20,41 @@
...
+        Establish an SSH connection to the master or agent nodes and list
```
Comment (albertostratio) [inline dcos_valkiria/cli.py]: I'd say: "Establish an SSH connection to the master or agent nodes and get the status of the Valkiria agent process on them."

File: README.rst [hunk 9]
```diff
@@ -1,7 +1,23 @@
 DCOS Valkiria Subcommand
 ==========================
 
-Valkiria DCOS subcommand
+The Valkiria DCOS subcommand allow you to install agents of https://github.com/Stratio/valkiria/ and do the next operations to iteract with them:
```
Comment (albertostratio) [inline README.rst]: there's a typo "iteract" should be "interact"

**Pull request merged**